src/terms/terms.jl

```diff
@@ -59,5 +59,13 @@ include("psp_correction.jl")
 include("entropy.jl")
 include("magnetic.jl")
 
+# breaks_symmetries on a term builder answers true if this term breaks
+# the symmetries of the lattice/atoms (in which case kpoint reduction
+# is invalid)
+breaks_symmetries(term_builder::Any) = false
+breaks_symmetries(term_builder::Magnetic) = true
+breaks_symmetries(term_builder::ExternalFromReal) = true
+breaks_symmetries(term_builder::ExternalFromFourier) = true
+
```

Comment:
I thought so too but it made more sense here: this is specialized enough that it makes sense to centralize the information, and we already have the include list above where we list all the terms. I would not docstring it because it's pretty internal, and docstrings act (to me) as a sign this is actually meant to be used.

Reply:
I would not go that far with docstrings. About the centralisation I'm not too sure. I'm thinking about someone who implements a new term type. It feels strange to be in the need to add information in 2 (3, 5?) different places rather than just copying a template file and modifying what's in there. The latter is much closer to what I would do and supports putting the specialisation together with the struct definition.

Reply:
In principle I agree with you, but in this specific instance it feels more natural to do it this way. You need to add the information in that file anyway because of the include, so might as well do it. On the other side, imagine trying to understand what that symmetry breaking is and having to find out which terms support it, you have to grep for it. Also I don't expect any other symmetry-breaking terms than the ones there.

Reply:
Ok that's a valid point. Let's keep it for now than.